src/components/widgets/Schedule.vue

```diff
@@ -541,7 +543,7 @@
                         class="timebar"
                         :style="timebarSubchildStyle(task, rootElement)"
                         :key="index"
-                        :title="`${task.entity.name} (${task.startDate.format('YYYY-MM-DD')} - ${task.endDate.format('YYYY-MM-DD')})`"
+                        :title="`${task.entity.name} (${task.startDate.format('YYYY-MM-DD')} - ${task.endDate.format('YYYY-MM-DD')}) ${formatDuration(task.estimation)} ${$t('schedule.md')}`"
```

Comment:
This line is way too long.

Reply:
I moved the title formatting into a method to improve readability.